src/bindings/c/src/common.h

```diff
@@ -37,6 +37,8 @@
         return ov_status_e::UNEXPECTED;                       \
     }
 
+#define PRINT_ERROR(str) std::cout << "Error: " << str << std::endl
```

Comment:
OpenVINO is a library. We should add nothing to standard output without customer request.

Reply:
ok, let's remove it.